packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -212,21 +201,38 @@ export class ElectronMainApplication {
     }
 
     async start(config: FrontendApplicationConfig): Promise<void> {
-        const args = this.processArgv.getProcessArgvWithoutBin(process.argv);
-        this.useNativeWindowFrame = this.getTitleBarStyle(config) === 'native';
-        this._config = config;
-        this.hookApplicationEvents();
-        this.showInitialWindow();
-        const port = await this.startBackend();
-        this._backendPort.resolve(port);
-        await app.whenReady();
-        await this.attachElectronSecurityToken(port);
-        await this.startContributions();
-        await this.launch({
-            secondInstance: false,
-            argv: args,
-            cwd: process.cwd()
-        });
+        const argv = this.processArgv.getProcessArgvWithoutBin(process.argv);
+        createYargs(argv, process.cwd())
+            .command('$0 [file]', false,
+                cmd => cmd
+                    .option('userDataArea', {
```

Comment:
I suggest using `electronUserData` to clarify that this is about the Electron user data, and not to confuse with the config location (`.theia` by default or defined by `$THEIA_CONFIG_DIR`). WDYT?